src/v/cluster/archival/archival_policy.cc

```diff
@@ -331,10 +327,10 @@ static ss::future<std::optional<std::error_code>> get_file_range(
             //
             // [...][10, 20][40, 50][...]
             // Target offset: 30
-            // Seek result offset: 21
```

Comment:
Oh really? It seems a little surprising, 11 seems a bit arbitrary, given it's with in a batch. Thought it'd be prev offset (20) + 1?

Reply:
Oops yeah. I re-worded this after reading some (buggy) test output. Thought I double checked it, but the original comment stands correct, reverted.